dashboard/app/views/home/_tos.md

```diff
@@ -1,19 +1,19 @@
-Date of Last Revision: March 1, 2017
+Date of Last Revision: October 25, 2017
 
-Code.org is a 501(c)(3) nonprofit organization (“Code.org”). Our mission is to give every student the opportunity to learn computer science. As a part of that mission, Code.org operates the web sites located at [https://code.org](https://code.org), [https://csedweek.org](https://csedweek.org), [https://computinginthecore.org](https://computinginthecore.org), [https://hourofcode.com](https://hourofcode.com), other websites, information, text, curricula, videos, graphics, photos, APIs, email notifications and other materials and related products and services (the “**Services**”), including the tutorial platform at [https://studio.code.org](https://studio.code.org) (“**Code Studio**”).
+Code.org is a 501(c)(3) non-profit organization (“Code.org”). Our mission is to give every student the opportunity to learn computer science. As a part of that mission, Code.org operates the web sites located at [http://code.org](http://code.org), [http://studio.code.org](http://studio.code.org), [http://csedweek.org](http://csedweek.org), [http://www.k12cs.org](http://www.k12cs.org), [http://hourofcode.com](http://hourofcode.com), [https://forum.code.org](https://forum.code.org), other websites, information, text, curricula, videos, graphics, photos, APIs, email notifications and other materials and related products and services (the “**Services**”).
```

Comment:
* web sites can be one word
* all of our URLs should be https
* curriculum.code.org?